.github/workflows/release-recover-id-object.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish Docker image for recover-id-object to DockerHub
+
+on:
```

Comment:
Suggestion: Include a manual trigger which we needed in the past sometimes when tagging was messed up.
```
  workflow_dispatch: # allows manual trigger
```

Reply:
I would like that, but for I consider that out of scope for this PR.

Scope for the PR is strictly to align it with the other releases. So if I did that I would also have to do that in the other releases. :)

Reply:
The `workflow_dispatch` is included in the set up of most other releases e.g:
https://github.com/Concordium/concordium-dapp-examples/tree/main/.github/workflows

Reply:
Adding it anyways